src/lib/stream-to-async-iterable.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+module.exports = function toAsyncIterable (res) {
+  const { body } = res
+
+  // An env where res.body getter for ReadableStream with getReader
+  // is not supported, for example in React Native
+  if (!body) {
+    if (res.arrayBuffer) {
```

Comment:
@achingbrain Do we want to `console.log` anything in this case, like "falling back to a hackier approach since the fetch being used here doesn't implement the streams spec"?